rj_gameplay/stp/action/__init__.py

```diff
@@ -83,4 +83,5 @@ def create(self, action: Type[ActionT]) -> ActionT:
 
 class Ctx:
     """Context for actions, dummy for now"""
-    ...
\ No newline at end of file
+    ...
+    
```

Comment:
Not blocking, but nit: in a bunch of places you added a new line with a bunch of extra indentation, not a trailing newline. As a result, all of these files still have the "no trailing newline" marker _and_ extra crap at the end.